core/src/runtime/mod.rs

```diff
@@ -838,169 +815,22 @@ pub mod tests {
   }
 
   #[test]
-  #[should_panic]
   fn test_panic() {
     let program = panic_program();
     let mut runtime = Runtime::new(program, None, AthenaCoreOpts::default(), None);
-    runtime.execute().unwrap();
-  }
-
-  #[test]
-  fn test_wallet() {
```

Comment:
Why remove this?

Reply:
Hi, I planned to explain in the description after fixing up the CI. Two reasons:
1. `core` is not the place for programs' tests
2. the same functionality is already tested in the tests in `examples/wallet/script/src/bin/execute.rs` (it's almost identical)